android/app/build.gradle

```diff
@@ -105,6 +105,16 @@ android {
             abiFilters "armeabi-v7a", "x86"
         }
     }
+    signingConfigs {
+        release {
+            if (project.hasProperty('MYAPP_RELEASE_STORE_FILE')) {
+                storeFile file(MYAPP_RELEASE_STORE_FILE)
+                storePassword MYAPP_RELEASE_STORE_PASSWORD
```

Comment:
Sorry you may have missed the link (click [here](https://pilloxa.gitlab.io/posts/safer-passwords-in-gradle/)). So I don't store my password inside the gradle file and use keychain access instead. _However_, there's no real way to include that logic into the placeholder I think so I'm okay with having this as a placeholder to be honest.